### ISSUE TITLE
Zoom is being laggy.

### ISSUE THREAD

**Issue by MubinMuhammad** [swaywm/sway#7969]
### Please read the following before submitting:
Zoom is being very buggy. There random windows popup ones in a while and menu doesn't open, sometimes they do, but they instantly goes away. Here is a video for clarity:

https://github.com/swaywm/sway/assets/101094719/a09829bb-616e-4f13-bd48-aba20bb4d4e3

### Please fill out the following:
  - **Sway Version:** sway version 1.8.1

- **Debug Log:**
  - [The log](https://gist.github.com/MubinMuhammad/a1188e5dc216bfa24a61fbf7a2245689)

- **Configuration File:**
```
# variables
set $mod   Mod4
set $left  h
set $down  j
set $up    k 
set $right l 
set $term  alacritty
set $menu  exec tofi-drun | xargs swaymsg exec --

# setting the wallpaper
# output * bg $HOME/.config/wallpapers/12.jpg fill

# common bindings
bindsym $mod+Return exec $term
bindsym $mod+d exec $menu
bindsym $mod+Shift+q kill
bindsym $mod+Shift+c reload
bindsym $mod+Shift+e exec swaynag -t warning -m 'Exit Sway?' -B 'Yes, exit sway' 'swaymsg exit' --font="terminess Nerd Font Bold"

floating_modifier $mod normal


# more bindings
bindsym $mod+$left        focus left
bindsym $mod+$down        focus down
bindsym $mod+$up          focus up
bindsym $mod+$right       focus right
bindsym $mod+Left         focus left
bindsym $mod+Down         focus down
bindsym $mod+Up           focus up
bindsym $mod+Right        focus right
bindsym $mod+Shift+$left  move left
bindsym $mod+Shift+$down  move down
bindsym $mod+Shift+$up    move up
bindsym $mod+Shift+$right move right
bindsym $mod+Shift+Left   move left
bindsym $mod+Shift+Down   move down
bindsym $mod+Shift+Up     move up
bindsym $mod+Shift+Right  move right
bindsym $mod+1 workspace number 1
bindsym $mod+2 workspace number 2
bindsym $mod+3 workspace number 3
bindsym $mod+4 workspace number 4
bindsym $mod+5 workspace number 5
bindsym $mod+6 workspace number 6
bindsym $mod+7 workspace number 7
bindsym $mod+8 workspace number 8
bindsym $mod+9 workspace number 9
bindsym $mod+0 workspace number 10
bindsym $mod+Shift+1 move container to workspace number 1
bindsym $mod+Shift+2 move container to workspace number 2
bindsym $mod+Shift+3 move container to workspace number 3
bindsym $mod+Shift+4 move container to workspace number 4
bindsym $mod+Shift+5 move container to workspace number 5
bindsym $mod+Shift+6 move container to workspace number 6
bindsym $mod+Shift+7 move container to workspace number 7
bindsym $mod+Shift+8 move container to workspace number 8
bindsym $mod+Shift+9 move container to workspace number 9
bindsym $mod+Shift+0 move container to workspace number 10
bindsym $mod+b splith
bindsym $mod+v splitv
bindsym $mod+s layout stacking
bindsym $mod+w layout tabbed
bindsym $mod+e layout toggle split
bindsym $mod+f fullscreen
bindsym $mod+Shift+space floating toggle
bindsym $mod+space focus mode_toggle
bindsym $mod+a focus parent
bindsym $mod+r mode "resize"

mode "resize" {
    bindsym $left resize shrink width 20px
    bindsym $down resize grow height 20px
    bindsym $up resize shrink height 20px
    bindsym $right resize grow width 20px

    bindsym Left resize shrink width 20px
    bindsym Down resize grow height 20px
    bindsym Up resize shrink height 20px
    bindsym Right resize grow width 20px

    bindsym Return mode "default"
    bindsym Escape mode "default"
}

bar {
  swaybar_command swaybar

  status_command /usr/bin/i3status-rs $HOME/.config/i3status/config.toml
  gaps 8 128
  position bottom
  font pango:FiraCode Nerd Font Bold 10

  colors {
    statusline #c5c8c6
    background #050505

    focused_workspace #8abeb7 #81a2be #1d1f21
  }
}

# startup executions
exec swww init

default_border pixel 2
default_floating_border pixel 2

# border colors
client.focused          #454545 #454545 #504945 #454545   #454545
client.focused_inactive #c5c8c6 #454545 #454545 #454545   #454545
client.unfocused        #504945 #454545 #504945 #504945   #504945
client.urgent           #fabd2f #fabd2f #fabd2f #fabd2f   #fabd2f
client.placeholder      #b8bb26 #b8bb26 #b8bb26 #b8bb26   #b8bb26
client.background       #928374

# swayfx config
# blur enable
# blur_xray enable
# blur_passes 2
# blur_radius 10
# layer_effects "panel" shadows enable, corner_radius 6;
# corner_radius 6
# default_dim_inactive 0.2
# shadows enable

input 2522:49418:A4Tech_USB_Mouse {
  accel_profile flat
}

# window gaps
gaps inner 4
gaps outer 4

# includes
include /etc/sway/config.d/*
```

- **Description:**
  - I tried to install zoom with AUR, that failed. Tried to install the `zoom-system-qt` package, that also did.

**Comment by bisby** [comment]
I also get this issue on the View button in a video chat that lets you swap between "Speaker View" or "gallery view", and also on the chat dropdown that lets you select a recipient for DMs.

**Comment by David96** [comment]
For me this was fixed by https://gitlab.freedesktop.org/wlroots/wlroots/-/merge_requests/4514
This fix is also included in the backports for wlroots 0.17.2 so if you don't want to use the main branch you might have to wait for that release.

**Comment by MubinMuhammad** [comment]
Ok, great to know that people are working in this!
